web/src/components/filings-calendar/FilingsCalendar.tsx

```diff
@@ -103,19 +103,19 @@ export const FilingsCalendar = (props: Props): ReactElement => {
     if (displayToggleButton) {
       return userData?.preferences.isCalendarFullView ? (
         <UnStyledButton
-          style="light"
-          noRightMargin
-          className="font-body-2xs padding-y-05 padding-x-1"
+          style={"transparentBgColor"}
+          isRightMarginRemoved
+          className="font-body-2xs padding-x-1 text-normal text-base border-1px border-base-light hide-unhide-button padding-y-11px"
           onClick={handleCalendarOnClick}
         >
           <Icon className="usa-icon--size-3 margin-right-05">list</Icon>
           {Config.dashboardDefaults.calendarListViewButton}
         </UnStyledButton>
       ) : (
         <UnStyledButton
-          style="light"
-          noRightMargin
-          className="font-body-2xs padding-y-05 padding-x-1"
+          style={"transparentBgColor"}
+          isRightMarginRemoved
+          className="font-body-2xs padding-x-1 text-normal text-base border-1px border-base-light hide-unhide-button padding-y-11px"
           onClick={handleCalendarOnClick}
```

Comment:
[Dust] might make sense to consolidate these 2 that are the same into a variable in this file (I know its not he point of the story though hence dust)